content/en/events/gsoc-2024.md

```diff
@@ -0,0 +1,219 @@
++++
+title = "Google Summer of Code 2024"
+description = "Google Summer of Code 2024"
++++
+
+---
+
+Kubeflow Community is excited to announce that we are applying this year to participate in the the [__Google Summer of Code 2024__](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
+
+
+Below are the "ideas/features" from different projects in the Kubeflow that being submitted for seeking proposals from GSoC Students.
+
+## Registration
+
+Please go to [Google Summer of Code 2024](https://buildyourfuture.withgoogle.com/programs/summer-of-code) and sign up as student and look up Kubeflow organization and select the projects you want to submit proposals. Note, you can submit more than one proposal. Contributor Applications open **March 20, 2024** and end on **Apr 4th 2024**. So start writing your proposals.
+
+## Mentors
+
+Kubeflow community is an unparalleled opportunity to make a lasting impact and share your wealth of experience. As a mentor, you'll have the chance to guide and shape the next generation of talent, while also collaborating with like-minded individuals to elevate projects to new heights. With dedicated support mechanisms like community recognition and feedback channels, you'll feel valued and empowered every step of the way. Join us in this exciting journey of mentorship and collaboration! If you want submit an project idea reach out Ramesh Reddy on slack.
+
+## Students
+
+Welcome to an incredible chance to dive into the vibrant Kubeflow community! Join us and become a valued contributor to our rapidly expanding network dedicated to AI/ML technologies on Kubernetes. Kubeflow mentors come with years of valuable experience and a passion for community growth and project enhancement, they're the guiding stars of our program. Immerse yourself in a dynamic environment where you'll not only gain insights from top industry experts but also sharpen your skills in open source collaboration, time management, and presentation prowess alongside making impactful technical contributions to projects of your interest. Our dedicated mentors will be by your side every step of the way, offering invaluable guidance and support throughout the project. Get ready to embark on an exciting journey of growth and learning!
+
+**If you need to reach out our mentors, join [Kubeflow Slack](https://kubeflow.slack.com/archives/C7REE0ETX) and look up your mentor or find project specific channel on Kubeflow slack**
```

Comment:
Probably use the Kubeflow slack website link (makes it easier for people to get invited by the bot).

https://www.kubeflow.org/docs/about/community/#kubeflow-slack

Reply:
agree, updated the link